pybamm/spatial_methods/spatial_method.py

```diff
@@ -169,7 +174,19 @@ def boundary_value(self, symbol, discretised_symbol, side):
         :class:`pybamm.Variable`
             The variable representing the surface value.
         """
-        raise NotImplementedError
+        n = 0
+        for domain in symbol.domain:
+            n += self.mesh[domain][0].npts
+        if side == "left":
+            left_vector = coo_matrix(([1], ([0], [0])), shape=(1, n))
```

Comment:
maybe a short comment on coo_matrix would be useful for explaining whats going on without needing to go find the coo_matrix documentation